lib/smart_proxy_ansible/playbooks_reader.rb

```diff
@@ -0,0 +1,31 @@
+module Proxy
+  module Ansible
+    class PlaybooksReader
+      class << self
+        def playbooks_names
+          ReaderHelper.collections_paths.split(':').map { |path| read_collection_playbooks(path).map { |playbook| playbook[:name] } }.flatten
+        end
+
+        def playbooks(playbooks_to_import)
+          ReaderHelper.collections_paths.split(':').reduce([]) do |playbooks, path|
+            playbooks.push(read_collection_playbooks(path, playbooks_to_import: playbooks_to_import))
```

Comment:
could we use `concat` instead of `push`? that would allow us to build the array directly and there would be no need for `flatten`ing.